src/__tests__/index.ts

```diff
@@ -192,6 +238,20 @@ describe('verifyPresentation', () => {
     expect(verified.verifiablePresentation).toBeDefined()
   })
 
+  it('rejects a Presentation without matching challenge', () => {
+    const options: VerifyPresentationOptions = {
+      challenge: 'TEST_CHALLENGE'
+    }
+    expect(verifyPresentation(PRESENTATION_JWT, resolver, options)).rejects.toThrow('Presentation does not contain the mandatory challenge (JWT: nonce) for : TEST_CHALLENGE')
+  })
+
+  it('rejects a Presentation without matching domain', () => {
+    const options: VerifyPresentationOptions = {
+      domain: 'TEST_DOMAIN'
+    }
+    expect(verifyPresentation(PRESENTATION_JWT, resolver, options)).rejects.toThrow('Presentation does not contain the mandatory domain (JWT: aud) for : TEST_DOMAIN')
+  })
+
   it('rejects an invalid JWT', () => {
     expect(verifyPresentation('not a jwt', resolver)).rejects.toThrow()
```

Comment:
Just applying the boy scout principle here :)

```suggestion
    expect.assertions(1)
    expect(verifyPresentation('not a jwt', resolver)).rejects.toThrow()
```

Reply:
this is not even my code. :) (and I think it's missing so far in general).

Reply:
Of course :)
I just took the opportunity to improve some nearby code, even if it's not related to this PR.
Feel free to ignore these unrelated suggestions, since it's a pattern that was missing from the tests anyway.